src/libponyc/codegen/genexe.c

```diff
@@ -294,7 +294,7 @@ static bool link_exe(compile_t* c, ast_t* program,
 
   snprintf(ld_cmd, ld_len,
 #if defined(PLATFORM_IS_ARM)
-    "%s -execute -arch %.*s "
+    "%s -execute -no_pie -arch %.*s "
```

Comment:
This option doesn't exist on the m1. I don't know if it's something related to arm64 or apple-specific.

Reply:
yup. it was a random test. you can ignore.